### DISCUSSION TITLE
Change the color of Switch toggler

### DISCUSSION THREAD

**Discussion by ChinonsoIg** [chakra-ui/chakra-ui#7364] [Q&A] [answered]
I am aware that we could use colorScheme with Switch to change the background color of the Switch component. However, the color of the toggler remains white. 
Is there a way to change the white color to another color?

**Comment by keithburgie** [comment, marked as answer]
Switch is a [multi-part component](https://chakra-ui.com/docs/styled-system/component-style#styling-multipart-components). You can follow the docs here to style it: https://chakra-ui.com/docs/components/switch/theming

**Reply by ChinonsoIg** [reply]
@keithburgie . Thank you so much for the assistance. It worked.
